package.json

```diff
@@ -272,7 +272,7 @@
 		"prebuild:packages": "npm run clean:packages && lerna run build",
 		"build:packages": "npm run --silent build:package-types && node ./bin/packages/build.js",
 		"build:plugin-zip": "bash ./bin/build-plugin-zip.sh",
-		"clean:package-types": "tsc --build --clean",
+		"clean:package-types": "tsc --build --clean; rm -rf packages/*/build-types",
```

Comment:
Does the order matter? Just asking because you suggested running `rm -rf packages/*/build-types` before this command to me

Reply:
No, it shouldn't matter.

Reply:
`rm -rf` does not work on Windows OS. Probably not many contributors are building Gutenberg on a Windows host OS, but is there a way to change the command to work on Windows/Mac/Lunux all?

Reply:
I noticed that `rimraf` is already added as a dependency. We can use this library. This library works in a cross-platform.

Reply:
Sure, I used `rm -rf` because I noticed it already in another script.